modules/rostests/apitests/gdi32/StretchBlt.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Tests for the StretchBlt API
+ * COPYRIGHT:   Copyright 2020 Doug Lyons (douglyons at douglyons dot com)
+ */
+
+#include <stdarg.h>
+#include <assert.h>
+#include <string.h>
+
+#include "ntstatus.h"
+#define WIN32_NO_STATUS
+#include "windef.h"
+#include "winbase.h"
+#include "winerror.h"
+#include "wingdi.h"
+#include "winuser.h"
+#include "mmsystem.h"
+#include "wine/winternl.h"
+#include "wine/test.h"
+
```

Comment:
Not sure all these header files are required, especially `ntstatus.h` and `mmsystem.h`, maybe it's possible to minimize inclusions.

Reply:
The "wine/test.h" is where the "OK" test comes from, so it is needed. I removed the other two. Thanks.